server/index.js

```diff
@@ -37,18 +35,23 @@ app.get('/answers', function (req, res) {
       res.send(error);
       console.error(error);
     })
-});
+})
 
 app.get('/currentProduct', (req, res) => {
-  console.log(req.query)
-  currentProduct(req.query.productNum, TOKEN)
+  currentProduct(req.query.productNum)
     .then(data => res.send(data))
     .catch(err => {
       res.send(err)
       console.log('err in currentProduct get server-side', err)
     })
 })
 
+app.get('/getMetadata', (req, res) => {
+  getMetadata(req.query.productNum)
+    .then(data => res.send(data))
+    .catch(err => {
+      res.send(err)
+      console.lof('err in getMetadata server-side', err)
```

Comment:
Spelling error on line 54 "console.lof"

Reply:
Good catch! Thank you! 

Updated and committed the update.